docker-compose.yml

```diff
@@ -23,6 +23,8 @@ services:
       DB_DATABASE: postgres
       DB_HOST: postgres
       DB_PORT: 5432
+      CHAIN_ENDPOINT: ws://host.docker.internal:9944
```

Comment:
(nit) I think NETWORK_ENDPOINT makes this more clear its an override for the config.